CONTRIBUTE.md

```diff
@@ -0,0 +1,56 @@
+
+# Run the demo environment 
+
+# Step 1: Setup and Compile
+
+* Compile the yard repository with **mvn clean install**
+    * https://github.com/kiegroup/yard
+* Compile this project **mvn clean install**
+* In order to run the Backstage plugin the repository with the Akrivis plugin is required
+  * Current Backstage plugins repository 
+    * https://github.com/kiegroup/backstage-plugins
```

Comment:
i think this fork should be also moved to kubesmarts org. KIEGROUP org is the legacy area and we all new work moved to KUBESMARTS so even this fork is good to have it there as used together with Akrivis. WDYT?

Reply:
We need to actually target either backstage/community-plugins or redhat-developer/rhdh-plugins before milestone 2 is complete. The pick will likely be rhdh-plugins since that is where orchestrator is and it (maybe) allows us to do releases in easier and in sync with the product needs. So this is where it currently is, but it will move and the link will change in the following 2 weeks.